newsletter/2020/09-18.html

```diff
@@ -0,0 +1,266 @@
+    <h3>
+      <a href="https://raytracing.github.io/" target="_blank"
+        rel="noopener noreferrer">
+        Ray Tracing in One Weekend
+      </a>
+    </h3>
+    <p>
+      Do you like digging into the guts of how things work? Do you want to involve yourself in game dev, whether for career reasons or for personal enrichment? Learn graphics by building your own basic path tracer in one weekend!
+    </p>
+    <sub>Suggested by Alexa</sub>
+
+    <hr class="separator" />
+    <h2>Tech Scene Awareness</h2>
+    <h3>
+      <a href="https://twitter.com/tesseralis/status/1293649007739191296" target="_blank"
+        rel="noopener noreferrer">
+        Trouble at GatsbyJS
+      </a>
+    </h3>
+    <p>
+      Nat Alison’s thread on Twitter about her time contracting with GatsbyJS shows a story of a company that, though it presents itself well and provides a decent product, is riddled with poor management, greed, and false ally-ship. It should be a reminder to us, as we continue to grow and continue in the tech industry, we can always do better. Remember to treat your coworkers/juniors/employees with respect, and communicate openly.
+    </p>
+    <sub>Suggested by Olivia</sub>
+
+    <hr class="separator" />
+    <h2>Alumni Spotlight</h2>
+    <p>
+      As this is an alumni newsletter, we would love to share spotlights of awesome things you are doing.
+    </p>
+    <h4>
+      Josef Butts - OS Graduate 09/2018
+    </h4>
+    <p>
+      Josef recently accepted a new position as Senior Software Engineer at <a
+        href="https://www.autodesk.com/" target="_blank" rel="noopener noreferrer">Autodesk</a> in Portland! This position comes after working as a contractor at
+      Nike for a year.
+    </p>
+    <h4>
+      Heather Haylett - OS Graduate 07/2020
+    </h4>
+    <p>
+      Heather recently had her first day at <a href="https://vouch.io/" target="_blank" rel="noopener noreferrer">Vouch.io</a> as a Junior React Native UI/UX
+      Developer! The company is small and supportive, and she's excited to be on the team.
+    </p>
+    <h4>
+      Olivia Sabo-Rush - OS Graduate 04/2017
+    </h4>
+    <p>
+      Liv recently left Operation Spark to join Chicago based company <a href="https://themomproject.com/" target="_blank" rel="noopener noreferrer">The Mom Project</a>. She accepted a position as a Mid-level Software Engineer, building a platform to help moms, dads, and allies return to the workforce!
+    </p>
+    <p>
+      If you’re doing cool stuff you would like us to spotlight, let us know in this Google form!
+    </p>
+    <h3>
+      <a href="https://forms.gle/hcF3e7uEFaF9bu3Z7"
+        target="_blank" rel="noopener noreferrer">
+        Submit yourself for the alumni spotlight
+      </a>
+    </h3>
+
+    <hr class="separator" />
+    <h2>Alumni Lunch - Virtual Edition</h2>
+    <p>
+      This month we’ll be trying our first virtual alumni lunch! On Thursday, August 20th at 12:30 CST we’ll get
```

Comment:
lol are we doing alumni lunch this month? if so, we should update the date, and if not, we should probably remove this

Reply:
Good catch! Thank you, updating the date and link